### PR TITLE
Fix warning dictpath

### PR DESCRIPTION
**Pull request by FloLey**
# Description

Fix warning dictpath as it should now be imported from inmanta

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [ ] Attached issue to pull request
- [ ] Changelog entry
- [x] Type annotations are present
- [x] Code is clear and sufficiently documented
- [ ] No (preventable) type errors (check using make mypy or make mypy-diff)
- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Correct, in line with design
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )

### CODE REVIEW

File: src/pytest_inmanta_lsm/lsm_project.py [hunk 1]
```diff
@@ -27,6 +27,12 @@
     "    - If you are using v2 modules: make sure the inmanta-module-lsm is installed in your venv."
 )
 
+# Try to import from inmanta.util.dict_path, if not available, fall back to the deprecated inmanta_lsm.dict_path
+try:
+    from inmanta.util import dict_path
+except ImportError:
+    from inmanta_lsm import dict_path
+
```
Comment (FloLey) [inline src/pytest_inmanta_lsm/lsm_project.py:35]: @sanderr is this what you meant with the try catch? also I don't think it deserves a change entry, does it?

Comment (sanderr) [reply src/pytest_inmanta_lsm/lsm_project.py:35]: Indeed. Also agreed on the change entry part. Did you verify that the latest iso6 has dict_path in inmanta-lsm?

**Review by sanderr — CHANGES_REQUESTED** [review]
Requesting changes for open discussion

File: src/pytest_inmanta_lsm/lsm_project.py (hunk 1 above)
Comment (FloLey) [reply src/pytest_inmanta_lsm/lsm_project.py:35]: You are talking about iso6-stable right? It does indeed have the dict path in inmanta-lsm

File: tests/utils.py [hunk 2]
```diff
@@ -61,9 +61,7 @@ def module_v2_venv(module_path: str) -> Iterator[env.VirtualEnv]:
             [
                 venv.python_path,
                 "-m",
-                "inmanta.app",
-                "-X",
-                "module",
+                "pip",
                 "install",
                 "-e",
```
Comment (FloLey) [inline tests/utils.py:66]: rerequesting review for this, just to make sure

**Review by sanderr — APPROVED** [review]

**Pull request merged**